src/System.Diagnostics.TraceSource/src/System/Diagnostics/Switch.cs

```diff
@@ -158,6 +171,10 @@ protected int SwitchSetting
             }
         }
 
+        virtual protected internal string[] GetSupportedAttributes() {
```

Comment:
According to the [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), visibility should be the first modifier, e.g.:

```c#
protected internal virtual string[] GetSupportedAttributes() => null;
```